### ISSUE TITLE
How to report a security issue

### ISSUE THREAD

**Issue by Kirill89** [pistacheio/pistache#1032]
Hi,
I'd like to report security about security issue I found in Pistache.
What is the proper way to do it?

**Comment by Tachi107** [comment]
Hi, as far as I know we currently don't have a security policy, but we definitely should define one. In the mean time, you could send an email to the current maintainers; you can find the addresses here: [google/oss-fuzz/projects/pistache/project.yaml](https://github.com/google/oss-fuzz/blob/c34c308faad86d154b52586ff66de8d77187cafd/projects/pistache/project.yaml)

We should also have a mailing list on Launchpad, but I don't manage it (nor how know how it works - @kiplingw is the one that created it)

**Comment by Kirill89** [comment]
Thanks! I sent the disclosure message to the email you pointed.

**Comment by Tachi107** [comment]
I'm reopening since we should really create a `SECURITY.md` file

**Comment by kiplingw** [comment]
Yeah we have a Launchpad mailing list but IIRC @dennisjenkins75 didn't like it. It can be noisy when there are issues with the PPA builders for different architectures.

What I'd like to see is the O/P post the email I received as a new issue here, but with some way to hide it temporarily from anyone who is not a maintainer until we get a fix out. But I don't know if GitHub has a way of doing that?

**Comment by Tachi107** [comment]
I think that GitLab has something like this, but GitHub unfortunately doesn't
